mirage/route-handlers/me.js

```diff
@@ -107,6 +107,15 @@ export function register(server) {
     return schema.crateOwnerInvitations.where({ inviteeId: user.id });
   });
 
+  server.get('/api/private/crate_owner_invitations', function (schema) {
+    let { user } = getSession(schema);
+    if (!user) {
+      return new Response(403, {}, { errors: [{ detail: 'must be logged in to perform that action' }] });
+    }
+
+    return schema.crateOwnerInvitations.where({ inviteeId: user.id });
```

Comment:
the new endpoint actually works a little bit different. it supports (and iirc requires) query parameters to determine which invites to show.